docker/windows/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export VERSION=$1
+docker build -f Dockerfile -t microsoft/iotedgedev:$1 -t microsoft/iotedgedev:$1-nanoserver -t microsoft/iotedgedev:latest-nanoserver .
```

Comment:
Let's tag with windows instead of nanoserver.  I doubt we'll ever have another version of windows container.

Reply:
Done